### PR TITLE
feat: option to start in SHUTDOWN

### PR DESCRIPTION
**Pull request by sander2**

### CODE REVIEW

File: crates/security/src/lib.rs [hunk 1]
```diff
@@ -90,14 +90,25 @@ pub mod pallet {
...
+            }
+        }
+    }
 
     #[pallet::genesis_build]
     impl<T: Config> GenesisBuild<T> for GenesisConfig {
         fn build(&self) {
+            Pallet::<T>::set_status(self.initial_status);
+
             Pallet::<T>::insert_error(ErrorCode::OracleOffline);
```
Comment (gregdhill) [inline crates/security/src/lib.rs:111]: Maybe only set this if the initial status is error?

**Review by gregdhill — CHANGES_REQUESTED** [review]
We're definitely gonna have some merge conflicts with the kusama chain spec, for now could you uncommit the `chain_spec.rs` changes and I'll include the relevant update in `mainnet_genesis`?

File: crates/security/src/lib.rs (hunk 1 above)
Comment (sander2) [reply crates/security/src/lib.rs:111]: Should still be OracleOffline, imo. After shutdown, sudo should change to Error, and then if the oracle is online, the oracle will clear the error and set the status to online

**Comment by sander2** [comment]
> We're definitely gonna have some merge conflicts with the kusama chain spec, for now could you uncommit the `chain_spec.rs` changes and I'll include the relevant update in `mainnet_genesis`?

done

**Review by gregdhill — APPROVED** [review]

**Pull request merged**